soccer/gameplay/skills/pivot_kick.py

```diff
@@ -172,5 +172,6 @@ def role_requirements(self):
             if self.use_chipper:
                 r.chipper_preference_weight = role_assignment.PreferChipper
             r.require_kicking = True
+            r.PositionCostMultiplier = 1.5
```

Comment:
Can you put a comment here noting why the position cost multiplier is overridden?